src/muya/lib/utils/exportStyledHTML.js

```diff
@@ -162,6 +165,15 @@ class ExportHTML {
     $(`.${CLASS_OR_ID['AG_ACTIVE']}`).removeClass(CLASS_OR_ID['AG_ACTIVE'])
     $(`[data-role=hr]`).replaceWith('<hr>')
 
+    // create heading id's
+    const headings = $(headingClassNames.join(', '))
+    if (headings.length) {
+      headings.each((i, h) => {
+        const heading = $(h)
+        heading.attr('id', `${heading.attr('id')} ${heading.attr('data-id')}`)
```

Comment:
Is there a space between `${...}` and `${...}`?
There should be no space in `id` attribute according to the standard https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id

Reply:
Yes that's right, but I merge two separate IDs together. E.g. consider `id="ag-0"` and `data-id="foo-bar"` then the result should be `id="ag-0 foo-bar"`. The element now has two IDs `ag-0` and `foo-bar`.

Reply:
I found description of attribute id as bellow:

>This attribute's value must not contain whitespace (spaces, tabs etc.). Browsers treat non-conforming IDs that contain whitespace as if the whitespace is part of the ID. In contrast to the class attribute, which allows space-separated values, **elements can only have one single ID**.

Reply:
OK then I'll overwrite the ID attribute with `data-id`.